ui/src/components/Rules/Rule/RuleCreator/CommunityRuleUpload/index.tsx

```diff
@@ -16,7 +16,7 @@ const CommunityRuleUpload = (props: ICommunityRuleUpload) => {
   const descriptionRef = useRef<any>()
   const uploadedByRef = useRef<any>()
   const [thanksModal, setThanksModal] = useState<boolean>(false)
-  const [failed, setFailed] = useState<boolean>(false)
+  const [failed, setFailed] = useState<string | false>(false)
```

Comment:
You can just set the default to an empty string

```suggestion
  const [failed, setFailed] = useState<string>('')
```

Reply:
Done